client/src/components/TOTP/Register.js

```diff
@@ -13,48 +14,217 @@ class Register extends Component {
     super(props);
 
     this.state = {
+      view: 'SCAN_CODE',
```

Comment:
Do you want to define these as constants at the top of the file? It probably doesn't make sense to export them though.